### PR TITLE
 Change testrpm.sh to reflect new openquake/base container

### PR DESCRIPTION
**Pull request by daniviga**
https://ci.openquake.org/job/builders/job/rpm-builder/125/ (RabbitMQ returns an error during the installation because of Docker, but that's ok)

### CODE REVIEW

File: helpers/testrpm.sh [hunk 1]
```diff
@@ -18,25 +18,23 @@
...
+COPR_REPO=openquake-stable
 
 checkcmd docker
 
 while (( "$#" )); do
     case "$1" in
         "-h")
             echo "Usage: $0 [-l] [-r]"
-            echo -e "\nOptions:\n\t-l: test RPM locally\n\t-r <reponame>: use a GEM COPR repo provided by <reponame> (default: openquake)"
+            echo -e "\\nOptions:\\n\\t-l: test RPM locally\\n\\t-r <reponame>: use a GEM COPR repo provided by <reponame> (default: openquake)"
```
Comment (nastasi-oq) [inline helpers/testrpm.sh]: What is the reason to double the escaping char ?

File: helpers/testrpm.sh [hunk 2]
```diff
@@ -18,25 +18,23 @@
...
 }
 
-CUR=$(pwd)
-BASE=$(cd $(dirname $0)/.. && /bin/pwd)
-COPR_REPO=openquake
+COPR_REPO=openquake-stable
 
 checkcmd docker
 
 while (( "$#" )); do
```
Comment (nastasi-oq) [inline helpers/testrpm.sh]: More compact but less readable than `[ "$#"  -gt 0 ]`, I think.

File: helpers/testrpm.sh (hunk 1 above)
Comment (daniviga) [reply helpers/testrpm.sh]: Backslash is literal in "\n". Prefer explicit escaping: "\\n". [SC1117]

File: helpers/makerpm.sh [hunk 3]
```diff
@@ -29,36 +29,30 @@ BRANCH='HEAD'
...
-            shift
-            ;;
-        *)
-            BRANCH="$1"
-            shift
             ;;
     esac
 done
+shift $((OPTIND -1))
+if [ "$1" ]; then BRANCH="$1"; fi
```
Comment (nastasi-oq) [inline helpers/makerpm.sh]: `test "$1" && BRANCH="$1"`
Or a multi-line `if` statement, what do you think about ?

**Comment by nastasi-oq** [comment]
LGTM (again).

**Pull request merged**